docs/site/home.html

```diff
@@ -15,8 +15,11 @@ <h1 class="logo"><img src="site/img/logo-full.svg" alt="Google Cloud Platform" /
       </div>
 
       <div class="quote-box--supplementary">
-        <h2>One-line npm install</h2>
-        <pre>$ npm install --save gcloud</pre>
+        <pre><code class="subtle--blue">$</code> npm install <code class="subtle--blue">--save</code> <strong>gcloud</strong></pre>
+        <h4 class="latest-release subtle" ng-if="latestRelease">
+          Latest Release <strong class="white">{{latestRelease}}</strong>
```

Comment:
I went with linking to the release page on GitHub, but if you think another option is better, it doesn't matter much to me. I personally never go to a package on npmjs.org, and if I do end up there, I quickly find the link to GitHub, as on GH, I get the same content with the benefit of code and an ultra-familiar interface. (Not counting the metadata from npm that I don't find too interesting)